src/content/promos.ts

```diff
@@ -42,7 +48,9 @@ export const promos: Promo[] = [
 		message: 'VueConf US 2025: Use code VUE25MEETJS for a special discount!',
 		cta: '👉 Redeem Voucher',
 		link: 'https://reg.connectevents.io/ConnectEvents/Vueconfus2025/redeem?voucher=VUE25MEETJS',
+		eventLink: 'https://vueconf.us',
 		expiresAt: '2025-05-21T23:59:59+02:00',
+		description: 'VueConf US is the premier Vue.js conference in North America, held in New Orleans on May 20-21, 2025. Join for two days of code playground, talks, and networking with the Vue community. Optional workshop day available on May 19. Save $199 with our exclusive discount!',
 		gradient: 'bg-gradient-to-r from-green-400 via-blue-500 to-purple-500',
```

Comment:
Could be an optional field and default to one random from a list of a few readymade gradients

Reply:
Yes, I had many problems with that because Tailwind removes unused colors and that is something I think about.